### ISSUE TITLE
爱可可微博热门分享(8.2)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#235]
No 1. <a href="https://weibo.com/1402400261/Oqm3ObUvt">由北邮老师团队为青少年精心打造的免费AI通识课《AI在大前》</a>
No 2. <a href="https://weibo.com/1402400261/Oqn6swJNv">GitHub Models：AI工程师的新一代工具，将行业领先的大型和小型语言模型直接带到GitHub上的超过1亿用户，让 GitHub 用户直接在平台上访问各种领先的大型语言模型和小型语言模型,包括 Llama、GPT系列、Phi等,用户可以在 playground 里免费试用这些模型</a>
No 3. <a href="https://weibo.com/1402400261/OqlIZzP5L">[CV] Generalized Out-of-Distribution Detection and Beyond ......</a>
No 4. <a href="https://weibo.com/1402400261/OqlNGlMeB">[CL] Apple Intelligence Foundation Language Models 网页链接 提出两种高效、可解释、安全的 ......</a>
No 5. <a href="https://weibo.com/1402400261/Oqcu2zIR3">#免费##抽奖# 携手@博文视点Broadview 送出3本《AI for Science：人工智能驱动科学创新》</a>
No 6. <a href="https://weibo.com/1402400261/Oqlwr5pct">[LG]《Deep Learning for Options Trading: An End-To-End Approach》</a>
No 7. <a href="https://weibo.com/1402400261/Oqciw3L8r">[LG] Machine Unlearning in Generative AI: A Survey 全面系统地总结和评价了生成式AI机器遗忘学习的技术发展现状、应用场景、评价体系、存在挑战和未来方向。 ......</a>
No 8. <a href="https://weibo.com/1402400261/OqqBVuWUE">ONNX-SAM2-Segment-Anything：用于图像和视频分割的ONNX模型，支持通过点和掩模进行自定义标注，适用于深度学习和计算机视觉领域的研究和应用</a>
No 9. <a href="https://weibo.com/1402400261/OqiUcwf3f">晚安～ [月亮]</a>
No 10. <a href="https://weibo.com/1402400261/OqcgicRV3">[LG] How to Choose a Reinforcement-Learning Algorithm 通过概述各类强化学习算法的特性，提供了算法选择的指南，以更系统有效的方式设计满足特定问题需求的强化学习方案。 ......</a>
No 11. <a href="https://weibo.com/1402400261/Oqn2PbVVU">自制YouTube视频摘要生成器：利用大型语言模型(LLM)和Python的yt-dlp工具，创建一个实用工具，自动从YouTube视频字幕中提取关键点和见解，帮助用户快速掌握视频内容，无需观看整个视频</a>
No 12. <a href="https://weibo.com/1402400261/OqnVCD7kh">LlamaVoice：基于Llama的大规模语音生成模型，提供推理和训练能力，采用直接预测连续特征的新颖方法，与传统依赖向量量化的离散语音代码预测模型相比，提供了更流畅高效的处理过程</a>
No 13. <a href="https://weibo.com/1402400261/OqdMMsjpe">“爱可可AI前沿快报”第31期：网页链接从零开始做播客整整一个月了，在小宇宙的订阅人数刚刚超过了700人，值得一个小小的庆祝。这几天学着用免费的Audacity录音和修剪，很享受这个制作的过程，常用的操作也从单一的删除变成了删除加静音，对发音和气息也有了新的认识，每天都有进步的感觉 ......</a>
No 14. <a href="https://weibo.com/1402400261/OqlvFhyY3">[CL]《Dancing in Chains: Reconciling Instruction Following and Faithfulness in Language Models》</a>
No 15. <a href="https://weibo.com/1402400261/Op8mBhmn0">[CL] A Comprehensive Survey of LLM Alignment Techniques: RLHF, ......</a>
No 16. <a href="https://weibo.com/1402400261/OqpgfnNJu">SegDrawer：支持语义分割和视频分割的简单静态网页绘图工具，采用交互式Segment Anything Model 2 (SAM2)</a>
No 17. <a href="https://weibo.com/1402400261/OqlqQd6m5">早！[太阳] ​</a>
No 18. <a href="https://weibo.com/1402400261/OqphexJYt">DistillKit：开源大型语言模型蒸馏工具包，旨在提供易于使用的工具进行研究、探索和增强开源大型语言模型（LLM）蒸馏方法的应用</a>
No 19. <a href="https://weibo.com/1402400261/Oqqzj7DAa">Clarity：AI研究的极简网站模板，专为展示人工智能研究成果设计，具有模块化和清晰的设计，易于定制，创建简单、基于项目的个人网站</a>
No 20. <a href="https://weibo.com/1402400261/OqlwBcAvR">提出一种端到端深度学习框架，通过直接从市场数据学习复杂的风险调整期权交易决策，在S&P100股票期权上的回测显示明显优于现有规则方法。//[LG]《Deep Learning for Options Trading: An End-To-End Approach》</a>
No 21. <a href="https://weibo.com/1402400261/Oq851gP1n">蚂蚁多模态框架：Ant Group多模态认知团队的研究成果集成，提供数据集管理、数据处理、训练流程、模型和模块的标准化多模态功能，支持自定义扩展</a>
No 22. <a href="https://weibo.com/1402400261/OqnNfrkhj">NotShazam：基于Shazam音乐识别算法的实现，集成Spotify和YouTube API进行音乐搜索和下载，支持音频指纹匹配技术</a>
No 23. <a href="https://weibo.com/1402400261/OqlCuEtuY">[LG]《MoMa: Efficient Early-Fusion Pre-training with Mixture of Modality-Aware Experts》</a>
No 24. <a href="https://weibo.com/1402400261/OqlZK7y8q">今日推介(第1485期)：用重复采样扩展推理计算、语言模型中的指令遵循与忠实性、基于深度学习的期权交易端到端方法、模态感知专家混合的高效早期融合预训练、基于LLM ......</a>
No 25. <a href="https://weibo.com/1402400261/Oqn0WjQ4D">爱可可AI前沿快报 Vol.32 网页链接做快报的一大初衷，是让青少年多接触一些一手的前沿技术信息，少一些二手的浮夸和危言耸听，用孩子们听得懂的语言让他们学会独立思考、尽早形成成自己的科研品味。</a>
No 26. <a href="https://weibo.com/1402400261/Oqcbi3Ako">[LG] Physics of Language Models: Part 2.1, Grade-School Math ......</a>
No 27. <a href="https://weibo.com/1402400261/Oq88flSnD">SSOReady：企业级单点登录（SSO）的开源开发工具，支持快速部署SAML认证，永久免费使用</a>
No 28. <a href="https://weibo.com/1402400261/OqlGhztZB">提出CLASI系统，使用LLM Agent和多模态训练方法，在实际场景数据集上实现人类同传水平的端到端汉英同声传译。//[CL]《Towards Achieving Human Parity on End-to-end Simultaneous Speech Translation via LLM Agent》</a>
No 29. <a href="https://weibo.com/1402400261/OqlG07qWV">[CL]《Towards Achieving Human Parity on End-to-end Simultaneous Speech Translation via LLM Agent》</a>
No 30. <a href="https://weibo.com/1402400261/Op4AnrD6y">SuperMemory：个人知识管理与内容整理的智能助手，支持Chrome扩展快速保存网页和推文，构建个人第二大脑</a>
No 31. <a href="https://weibo.com/1402400261/Oqlw7sr1e">通过两阶段微调实验证明了指令微调语言模型在遵循指令和保持忠实性之间存在明显权衡，并提出RESET方法有效调和两个训练目标的冲突，取得了在两个指标上都显著提升的结果。//[CL]《Dancing in Chains: Reconciling Instruction Following and Faithfulness in Language Models》</a>
No 32. <a href="https://weibo.com/1402400261/OqlCC2T8X">通过提出模态感知的专家混合模块MoMa，实现了混合模态早期融合语言模型的高效预训练。//[LG]《MoMa: Efficient Early-Fusion Pre-training with Mixture of Modality-Aware Experts》</a>
No 33. <a href="https://weibo.com/1402400261/Oq7XPrkia">MindSearch：基于大型语言模型的多智能体网络搜索引擎框架，性能类似Perplexity.ai Pro和SearchGPT，支持开源和闭源大型语言模型</a>
No 34. <a href="https://weibo.com/1402400261/Oqpi3ETar">Cold Compress：开源轻量工具包，用于创建和基准测试基于GPT-Fast的缓存压缩方法，支持自定义和扩展KV缓存压缩技术</a>
No 35. <a href="https://weibo.com/1402400261/OqcdKedfr">[LG] Can LLMs be Fooled? Investigating Vulnerabilities in LLMs ......</a>
No 36. <a href="https://weibo.com/1402400261/Oq82z55ry">大型语言模型提示库：为大型语言模型设计的提示库，提供模板装饰器和Jinja模板渲染功能，简化与语言模型的交互过程</a>
No 37. <a href="https://weibo.com/1402400261/OqlTkCTn5">[CL] Promises and Pitfalls of Generative Masked Language Modeling: ......</a>
No 38. <a href="https://weibo.com/1402400261/Oqm1d9kUS">//@爱可可-爱生活：欢迎参与~//#免费##抽奖# 携手@博文视点Broadview 送出3本《AI for Science：人工智能驱动科学创新》</a>
No 39. <a href="https://weibo.com/1402400261/OqnKF86kx">nanoGPT风格版本的Llama 3.1：简化的深度学习模型实现，致力于在单GPU上高效运行大规模语言模型，优化依赖并提供基础训练功能</a>
No 40. <a href="https://weibo.com/1402400261/Oq7ZJD3k3">ComfyUI-segment-anything-2：基于SAM 2模型的UI组件，用于实现图像分割功能，支持自动下载模型，简化图像编辑流程</a>
No 41. <a href="https://weibo.com/1402400261/Oq2YJgXrS">[LG] SAPG: Split and Aggregate Policy Gradients 提出一种新型基于策略的强化学习算法SAPG，通过分块学习多个策略并采用重要性采样聚合数据的方式，可以在大规模并行环境中获得比PPO等算法更好的性能。 ......</a>
No 42. <a href="https://weibo.com/1402400261/Oq9A4qQav">晚安～ [月亮] ​</a>
No 43. <a href="https://weibo.com/1402400261/OoZ4Z8K6n">[IR] Efficient Retrieval with Learned Similarities 证明MoL是习得相似度的通用近似表示，并提出相关的准确及近似top-K算法，在推荐检索任务上达到最先进水平和91倍的加速比。 ......</a>
No 44. <a href="https://weibo.com/1402400261/OfNPPruLU">流畅阅读：基于上下文语境的人工智能翻译引擎，为网站提供友好的翻译，提供母语般的阅读体验，支持多种翻译模式和快捷方式</a>
No 45. <a href="https://weibo.com/1402400261/OfE8ZDDns">Chat2API：一个将网页版ChatGPT转换为OpenAI API格式的服务，支持多账号轮询和多种GPT模型对话，无需账号即可使用免费、无限的GPT-3.5</a>
No 46. <a href="https://weibo.com/1402400261/Opr7LB4sC">[CL] OpenDevin: An Open Platform for AI Software Developers ......</a>
No 47. <a href="https://weibo.com/1402400261/Oqc58gWCN">[LG]《A Method for Fast Autonomy Transfer in Reinforcement Learning》</a>
No 48. <a href="https://weibo.com/1402400261/O2nnoEWHX">TableQAKit: 用于表格问答的工具包，支持LLM模型，提供可扩展的设计、全面的数据集和强大的方法，支持LLM的提示和微调方法、统一的数据接口、可复现的SOTA方法以及高效的LLM评估</a>
No 49. <a href="https://weibo.com/1402400261/OpurG997D">《机器学习中的不平衡数据》随书代码</a>
No 50. <a href="https://weibo.com/1402400261/Oqc8CyP2r">[AS]《Futga: Towards Fine-grained Music Understanding through Temporally-enhanced Generative Augmentation》</a>
